Tests/scripts/hook_validations/tests/integration_test.py

```diff
@@ -648,34 +648,6 @@ def test_is_default_arguments_non_default():
     assert validator.is_default_arguments() is False, \
         "The integration validator did not find invalid arg (needed to be default and not required)"
 
-
-def test_is_default_arguments_is_required():
-    validator = IntegrationValidator("temp_file", check_git=False)
-    validator.current_integration = {
-        "script": {
-            "commands": [
-                {
-                    "name": "domain",
-                    "arguments": [
-                        {
-                            "name": "domain",
-                            "required": True,
-                            "default": True
```

Comment:
@guyfreund no need to delete the test, modify it accordingly

Reply:
The test checks that if given required as true, then the file validation fails
Since we removed that condition, the test is not needed

Reply:
Fixed.